File: Python/백준/Bronze/11050. 이항 계수 1/이항 계수 1.py
def fac(n):
    if n == 0:
        return 1
    else:
        return n*fac(n-1)

def bc(n,k):
    return fac(n)//(fac(k)*fac(n-k))

n, k = map(int,input().split())
print(bc(n,k))

File: Java/백준/Bronze/11050. 이항 계수 1/이항 계수 1.java
import java.util.*;
import java.io.*;

class Main {
	public static void main(String args[]) throws Exception {
		Scanner sc = new Scanner(System.in);
		int N = sc.nextInt();
		int K = sc.nextInt();
		System.out.println(fac(N)/(fac(K)*fac(N-K)));
	}
	public static int fac(int n) {
		while (n>1) {
			return n*fac(n-1);
		}
		return 1;
	}
}